### DISCUSSION TITLE
Welcome to Matplot++ Discussions!

### DISCUSSION THREAD

**Discussion by alandefreitas** [alandefreitas/matplotplusplus#64] [General]
## 👋 Welcome to the Matplot++ community!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Help others.
  * Welcome others and are open-minded. 

Remember that this is a community we build together 💪.

**Comment by Ricardo-Oliv** [comment]
Hi, 
I am building this with Python 3.6.8 x64, everything builds ok. 

The examples don't run, I receive the following message: " 'gnuplot' is not recognized as an internal or external command,"

What am I missing?

Thanks  in advance for the help.

**Reply by alandefreitas** [reply]
It seems like you just need to install gnuplot 5.2.6+. You can download it from http://www.gnuplot.info or you can try:

```bash
sudo apt update
sudo apt install gnuplot
```

on Linux, or

```bash
brew install gnuplot
```

on Mac OS.

If you intend to use it from Python, [matplotlib](https://matplotlib.org) is a much more mature library though.

**Comment by Ricardo-Oliv** [comment]
Hi Alan,
thank you for your reply. It still did not solve my issue.

I am building a  C++ application and thought of using matplotplusplus to
make the charts.
However I am surely missing some early step, as I still receive the message
with the gnuplot installed.

I am using:

Python 3.6.8 x64
Visual Studio 2019 latest patch
gnuplot 5.2.8
CMake GUI 3.16.2

I build all projects with Visual Studio compiler with zero errors.

Thank you,
Ricardo Oliveira


Alan de Freitas <notifications@github.com> escreveu no dia domingo,
27/09/2020 à(s) 01:22:

**Reply by alandefreitas** [reply]
I'm not sure I understand how you are integrating that with Python. Is the C++ code working OK on its own? How is Python involved in what you are doing? 

I don't understand what you are trying to accomplish. What do you mean you are building it with Python 3.6.8 x64? Are you rewriting the building system? Or are you creating bindings for Python? 

Btw, would you mind opening a new discussion just for this with this link: https://github.com/alandefreitas/matplotplusplus/discussions/new?category_id=31250560? It would be easier to organize our thoughts and for the community to help in a dedicated discussion. Thanks...
